doc/release-notes/6.2025.1_update.md

```diff
@@ -0,0 +1,206 @@
+- Payara application server has been upgraded to version 6.2025.1.
+
+## Installation
+
+The following assumes the standard instructions to undeploy v6.5, stop payara, `export PAYARA=/usr/local/payara6`, etc. have been done already.
+
+### Upgrade to Payara 6.2025.1
+
+Option 1: 
+
+The steps below reuse your existing domain directory with the new distribution. You may also want to review the Payara upgrade instructions as it could be helpful during any troubleshooting:
+[Payara Release Notes](https://docs.payara.fish/community/docs/6.2025.1/Release%20Notes/Release%20Notes%206.2025.1.html).
+We also recommend you ensure you followed all update instructions from the past releases regarding Payara.
+(The most recent Payara update was for [v6.3](https://github.com/IQSS/dataverse/releases/tag/v6.3).)
+
+Move the current Payara directory out of the way:
+
+```shell
+mv $PAYARA $PAYARA.6.2024.6
+```
+
+Download the new Payara version 6.2025.1 (from https://www.payara.fish/downloads/payara-platform-community-edition/), and unzip it in its place:
+
+```shell
+cd /usr/local
+unzip payara-6.2025.1.zip
+```
+
+Replace the brand new `payara/glassfish/domains/domain1` with your old, preserved domain1:
+
+```shell
+mv payara6/glassfish/domains/domain1 payara6/glassfish/domains/domain1_DIST
+mv payara6-2024.6/glassfish/domains/domain1 payara6/glassfish/domains/
+```
+
+Deploy/install as normal
+
+Option2 : Modify the default 6.2025.1 domain1 configuration for your Dataverse instance. (Removes obsolete options, gets updated certificate store, makes tracking your Dataverse-specific changes relative to the the Payara default easier.)
+
+1. Stop Payara 6, if running.
+
+    ``
+    sudo -u dataverse /usr/local/payara6/bin/asadmin stop-domain
+    ``
+1. Move your existing payara installation to a backup directory
+
+```shell
+mv $PAYARA $PAYARA.6.2024.6
+```
+
+1. Download Payara 6.2025.1 and unzip it
+
+```shell
+    cd /usr/local
+    curl -L -O https://nexus.payara.fish/repository/payara- community/fish/payara/distributions/payara/6.2025.1/payara-6.2025.1.zip
+    unzip payara-6.2025.1.zip
+```
+
+1. Change ownership of the unzipped Payara to your "service" user ("dataverse" by default)
+
+   `sudo chown -R dataverse /usr/local/payara6`
+   
+1. Create a Dataverse-specific domain.xml file
+
+    i. Make a copy of the default domain.xml file (to be able to compare against later)
+
+   ``sudo -u dataverse cp /usr/local/payara6/glassfish/domains/domain1/config/domain.xml /usr/local/payara6/glassfish/domains/domain1/config/domain.xml.orig``
+   
+    ii. Retrieve the domain.xml file for this version of Dataverse
+
+```shell
+    cd /usr/local
```

Comment:
If this option is kept, the cd should be to `/usr/local/glassfish/domains/domain1/config`
and curl should have the `-L -O` options